setup/scaling/caching/configure-redis-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Configure Redis Usage"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker relies on its Redis cache for a number of reasons: caching
+infrastructure, storing live executions, returning pipeline definitions faster,
+etc... Each of these can be configured to your needs, whether it is to make
+Spinnaker more responsive, or reduce the load on a downstream dependency.
+
+## Configure infrastructure caching
+
+By default, Spinnaker's Clouddriver service will spawn "caching agents" every
+30 seconds. These agents are scheduled and coordinated across all instances of
+Clouddriver, allowing you to reduce the load on any single instance of
+Clouddriver by creating more replicas of the service.
+
+To adjust how this caching happens, Clouddriver exposes a few properties that
+can be set in `~/.hal/$DEPLOYMENT/profiles/clouddriver-local.yml`:
+
+```yaml
+# How many seconds (default 30) between runs of agent. Lowering this number
+# means the resources in the Spinnaker UI will be updated more frequently,
+# at the cost higher API/quota usage of your cloud provider.
+redis.poll.intervalSeconds:
+
+
+# How many seconds (default 600, 5 minutes) Clouddriver will wait to reschedule
+# an agent that never completes (never throws an error or returns cache data).
+# If your agents are taking a long time to complete their cache cycles
+# successfully and Clouddriver is prematurely rescheduling them, you can try to
+# raise this number.
+redis.poll.timeoutSeconds:
+```
+
+> `$DEPLOYMENT` is typically `default`. See [the
+> documentation](/reference/halyard#deployments) for more details.
+
+## Configure pipeline cleanup
+
+If you see that your Redis memory usage is growing unbounded, it's most likely
+due to the fact that the orchestration engine stores all pipeline executions
+forever by default. This can be configured by setting the following properties
+in `~/.hal/$DEPLOYMENT/profiles/orca-local.yml`:
+
+```yaml
+pollers:
+  oldPipelineCleanup:
+    enabled: true       # This enables old pipline cleanup.
```

Comment:
s/pipline/pipeline/

Are the comments in these yaml files written by you, on this occasion, or were they already there?

Reply:
These yaml files exist here only